### PR TITLE
save babystepZ to eeprom

### LINKED ISSUE #1293: Babystepping settings arent stored permanently
The save button doesn't execute M500 command to store the offset. It is only stored until the printer is restarted, so the save button has no use.

1. Start print
2. Go to adjust
3. Select babystepping
4. Adjust settings
5. Go back
6. Click save

**Expected behavior**
The adjusted offset is stored permanently using the M500 command.

**General Information:**
 - Raspberry Pi 3 B +
 - OctoDash V2.1.2

### PR DESCRIPTION
**Pull request by pciavald**
send M500 after setting babysetpZ to save the setting across printer reboots

Fixes #1293

### CODE REVIEW

File: src/app/print-control/print-control.component.ts [hunk 1]
```diff
@@ -238,6 +238,8 @@ export class PrintControlComponent implements OnInit, OnDestroy {
     // gotta love JS for that one.
     this.zOffset = Math.round((this.zOffset + value) * 100) / 100;
     this.printerService.executeGCode(`${this.configService.getZBabystepGCode()}${value}`);
+    // save babystepZ to EEPROM
+    this.printerService.executeGCode('M500')
```
Comment (UnchartedBull) [inline src/app/print-control/print-control.component.ts:242]:
```suggestion
    this.printerService.executeGCode('M500');
```

to make the linter happy :)

**Review by UnchartedBull — APPROVED** [review]
Looks good from my side. Fixes #1293.

**Comment by pciavald** [comment]
I noticed the Fixes link did not work from your comment, next time if i didn't know that i'm fixing a previous issue don't hesitate to edit the PR message!

**Comment by UnchartedBull** [comment]
Ok thank you for all your contributions! :) Just waiting for the pipeline to go green now.

**Comment by pciavald** [comment]
No problem, thank you for this amazing project, and allowing us to build upon it!

**Pull request merged**